packages/variants/src/variants.ts

```diff
@@ -219,7 +262,24 @@ export function getStrongVariant(theme: IPartialTheme): ITheme {
     // inputBackgroundCheckedHovered: p.themePrimary,
     inputForegroundChecked: p.themeDark,
     // inputFocusBorderAlt: p.themePrimary,
-    variantBorder: p.themeDark
+    variantBorder: p.themeDark,
+
+    buttonBackground: p.white,
+    buttonText: !fullTheme.isInverted ? p.themePrimary : p.neutralPrimary,
+    buttonBorder: 'transparent',
+    buttonBackgroundHovered: !fullTheme.isInverted ? p.themeLighter : p.themeLight,
+    buttonTextHovered: !fullTheme.isInverted ? p.themePrimary : p.neutralPrimary,
+    buttonBackgroundPressed: !fullTheme.isInverted ? p.themeLight : p.themeTertiary,
+    buttonTextPressed: !fullTheme.isInverted ? p.themePrimary : p.neutralPrimary,
+    buttonTextDisabled: !fullTheme.isInverted ? p.themeLight : p.themeTertiary,
+    buttonBorderDisabled: !fullTheme.isInverted ? p.themeLight : p.themeTertiary,
+    primaryButtonBackground: p.themePrimary,
```

Comment:
is this right for strong variant? shouldn't the normal button be p.themePrimary background so that it fades into the background?